src/bot.js

```diff
@@ -57,89 +61,52 @@ client.on('messageCreate', (message) => {
+  if (command === 'get-comic') {
     (async () => {
-      const file = [];
+      const comicId = args[0];
 
-      const feed = await parser.parseURL(RSS_URL);
-      feed.items.forEach((item) => {
-        const res = item.content.match(/(?<=src=").*\.(jpg|jpeg|png|gif)/gi);
-        file.push(res);
-      });
-      file.forEach((f) => {
-        message.channel.send({
-          content: `<${f}>`,
-          files: f,
-        });
-      });
+      const res = JFormatter((await xkcdService.getComicById(comicId)).data);
+
+      await PostComicToDiscord(client, rChannel, res);
     })();
   }
 
-  // TODO: remove this
-  if (command === 'cachefeed') {
+  // Check feed
+  if (command === 'feed') {
     (async () => {
-      const newItems = await feedchecker.checkFeed(RSS_URL);
-      if (newItems.length === 0) {
-        console.debug('no new items');
-        return;
-      }
-
-      console.debug('formatter', newItems[0]);
-
-      // There should only be one item if checkFeed is regularly scheduled.
-      const formattedComic = formatter(newItems[0]);
-      await botChannel.send({
-        content: `New xkcd posted!\n<${formattedComic.Url}>\n**${formattedComic.Title}**\n\`\`\`${formattedComic.Alt_text}\`\`\``,
-        files: [{
-          attachment: `${formattedComic.Img}`,
-          description: `${formattedComic.Alt_text}`,
-        }],
+      const feed = await parser.parseURL(RSS_URL);
+      feed.items.forEach((item) => {
+        const res = Formatter(item);
+        PostComicToDiscord(client, rChannel, res);
       });
     })();
   }
 });
 
+// Start message
 client.on('ready', (c) => {
-  c.channels.cache.get('943717767687864341').send('Hello!');
+  c.channels.cache.get(channelId).send('Hello there!');
 });
 
-// xkcd updates MWF in the evening PST. Exact update schedule varies.
-// Schedule the job to run MWF, every 30 minutes.
-const xkcdJob = new cron.CronJob('00 00,30 * * * 1,3,5', (() => {
+// Create new job which is supposed to run at 20:25:00 everyday.
+const xkcdJob = new cron.CronJob('15 52 16 * * *', (() => {
   (async () => {
-    const newItems = await feedchecker.checkFeed(RSS_URL);
-    if (newItems.length === 0) {
-      console.debug('no new items');
-      return;
-    }
-    // There should only be one item if checkFeed is regularly scheduled.
-    newItems.forEach(async (item) => {
-      const formattedComic = formatter(item);
-      await client.channels.cache.get('943717767687864341').send({
-        content: `New xkcd posted!\n<${formattedComic.Url}>\n**${formattedComic.Title}**\n\`\`\`${formattedComic.Alt_text}\`\`\``,
-        files: [{
-          attachment: `${formattedComic.Img}`,
-          description: `${formattedComic.Alt_text}`,
-        }],
-      });
-    });
+    const feed = await parser.parseURL(RSS_URL);
+    // const res = feed.items[0].content.match(/(?<=src=).*\.(jpg|jpeg|png|gif)/gi);"
+    const res = Formatter(feed.items[0]);
+
+    PostComicToDiscord(client, channelId, res);
```

Comment:
I'd really prefer if this used the rss feed checker. or at least, if it checks against a db whether or not an item is found. This way, the call is idempotent - this job can be run an infinite number of times, but will only take action if a NEW item is found.

Reply:
Will be done later down the line. Currently testing without involving RSS Feed Checker. You can change it as nessary.